model/AI/Properties/standardsCompliance.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# standardsCompliance
+
+## Summary
+
+Field captures what standards are being complied with. 
+
+## Description
+
+Text field that captures the list of standards that the AI software complies with. This could include both published and unpublished standards including ISO, IEEE, ETSI standards. These standards could (but not neccessarily) be used staisfy to a legal or regulatory requirement. Each standard is listed on a separate line or in a single line with each standard being comma separated. 
+
+## Metadata
+
+- name: standardsCompliance
+- Nature: DataProperty
+- Range: xsd:string
```

Comment:
Since this is in a list, standardsComplianceComment wouldn't work.  If we can do an enumeration that would be ideal.  If not, perhaps we just go with free form text.

Reply:
From discussions with Karen, free form text is our best option at this point.